packages/react-server/src/ReactFlightServer.js

```diff
@@ -117,6 +118,8 @@ import binaryToComparableString from 'shared/binaryToComparableString';
 
 import {SuspenseException, getSuspendedThenable} from './ReactFlightThenable';
 
+initAsyncDebugInfo();
```

Comment:
The explicit init here is really only due to the module cycle between the implementation and the config since we need `createAsyncHook` to have been resolved already.